app/login.html

```diff
@@ -0,0 +1,114 @@
+                            <li class="menu__item"><a class="menu__item-link" href="index.html">Home</a></li>
+                            <li class="menu__item"><a class="menu__item-link" href="#">About us</a></li>
+                            <li class="menu__item"><a class="menu__item-link" href="shop.html">Shop</a></li>
+                            <li class="menu__item"><a class="menu__item-link" href="#">Blog</a></li>
+                            <li class="menu__item"><a class="menu__item-link" href="#">Contacts</a></li>
+                            <li class="menu__item"><a class="menu__item-link" href="#">How it Works</a></li>
+                        </ul>
+                    </nav>
+                </div>
+                <div class="header__buttons">
+                    <a class="button header__button" href="login.html">Login</a>
+                    <a class="button header__button" href="register.html">Register</a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main>
+        <div class="modal container">
+            <div class="modal__panel">
+                <div class="tabs modal__tabs">
+                    <ul class="tabs__nav flex flex_center">
+                        <li class="tabs__item">
+                            <a class="tabs__link" href="#login">Login</a>
+                        </li>
+                        <li class="tabs__item">
+                            <a class="tabs__link tabs__link--active" href="#register">Register</a>
+                        </li>
+                        <li class="tabs__devider"></li>
+                    </ul>
+                    <div class="tabs__body">
+                        <div class="tabs__content" id="login">
+                            <div class="tabs__title text-center">Log In</div>
+                            <form class="flex flex_center" action="#" method="get">
+                                <div class="form__login">
+                                    <div class="form__controls">
+                                        <p class="form__login-title">E-mail</p>
+                                        <input class="form__input" minlength="5" type="text" name="email" required>
+                                    </div>
+                                    <div class="form__controls">
+                                        <p class="form__login-title">Password</p>
+                                        <input class="form__input" required type="password" name="Password"
+                                            id="Password">
+                                    </div>
+                                    <div class="form__forgot">
+                                        <a class="form__forgot-link" href="#">Forgot your password?</a>
+                                    </div>
+                                    <div class="form__controls form__controls_checkbox">
```

Comment:
![Screenshot from 2021-07-04 17-05-56](https://user-images.githubusercontent.com/19241160/124388130-1fdd8000-dcea-11eb-9a28-31292861b07f.png)

elements are not aligned